src/main/java/Application.java

```diff
@@ -62,10 +62,10 @@ private static void interactiveCSV(String[] args) throws Exception {
     }
 
     private static void printUsage() {
-        System.out.println("""
-                Usage: java Application <config-path> <mapping-path>
-                       java Application --csv <config-path> <data-path> <mapping-path> [--no-headers]
-                       java Application --i <config-path>
-                       java Application --csv --i <config-path> <data-path> [--no-headers]""");
+        System.out.println(
+                "Usage: java Application <config-path> <mapping-path>\n" +
```

Comment:
```suggestion
                "Usage: java Application <config-path> <mapping-path>\n" +
```
                "Usage: java -jar Application <config-path> <mapping-path>\n"